### PR TITLE
fix: Added second example to APIs tagged with property

### PR DESCRIPTION
**Pull request by kevinshahfws**

### CODE REVIEW

**Review by jlacivita — APPROVED** [review]

**Review by jlacivita — CHANGES_REQUESTED** [review]

File: src/sdks/manage/sdk.config.json [hunk 1]
```diff
@@ -115,19 +115,12 @@
...
             ],
             "provide": [
-                "xrn:firebolt:capability:token:session"
-            ]
-        },
-        {
-            "module": "Authentication",
-            "use": [
-                "xrn:firebolt:capability:token:account",
                 "xrn:firebolt:capability:token:session",
```
[the remark this answers is not in the document]
Comment (kevinshahfws) [reply src/sdks/manage/sdk.config.json]: addressed

**Review by kpears201 — APPROVED** [review]

**Pull request merged**